### PR TITLE
Ignore bundle when linting

### PR DESCRIPTION
**Pull request by andrei-picus-hs**

### CODE REVIEW

**Comment by andrei-picus-hs** [comment]
@flore77 please review.

**Comment by flore77** [comment]
LGTM!

**Comment by flore77** [comment]
Now is it LGTM :smile:

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ui/
+tests/
+.gitignore
```
Comment (andrei-picus-hs) [inline .npmignore]: This is automatically excluded.

Comment (flore77) [inline .npmignore]: Hmm ok. I didn't understand this from the [docs](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package).

**Pull request merged**